types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -121,9 +133,15 @@ const MuiCustomTable: React.FC<Props> = props => {
                     console.warn('action not handled.');
             }
         },
+        setFilterChipProps: () => {
+            return {
+                color: "secondary",
+                variant: "outlined"
+            };
+        },
         textLabels: {
             body: {
-                noMatch: 'Sorry, no matching records found',
+                noMatch: <p>Sorry, no matching records found</p>,
```

Comment:
```suggestion
                noMatch: 'Sorry, no matching records found',
```

`noMatch` should still be able to take a string so if you want to test that it can also take a ReactNode then please write a seperate test case

Reply:
something like "const noMatch: string | React.React.Node = ..." or would you prefer that I create a seperate table for this?

Reply:
separate table